### PR TITLE
Add elthosan's file

### PR DESCRIPTION
**Pull request by Allaoui-SA**
created a branch, created a file and made a commit, and opened a pull request

### CODE REVIEW

**Comment by elthosan** [comment]
test

**Review by Allaoui-SA — COMMENTED** [review]
add description

File: _posts/0000-01-02-elthosan.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
+Use the left arrow to go back!
```
Comment (Allaoui-SA) [inline _posts/0000-01-02-elthosan.md:6]: created a branch, created a file and made a commit, and opened a pull request

**Review by Allaoui-SA — COMMENTED** [review]
created a branch, created a file and made a commit, and opened a pull request

**Pull request merged**